front/src/modules/rollingStock/helpers/utils.ts

```diff
@@ -375,3 +469,86 @@ export const splitRollingStockProperties = (
   });
   return groupBy(displayedProperties, 'side') as { [key: string]: SchemaProperty[] };
 };
+
+/**
+ * According the conversion factors schema, convert a value if
+ * the factors exist in the schema
+ */
+export const convertUnits = (prevUnit: string, newUnit: string, prevValue: number): number => {
```

Comment:
```suggestion
type Unit =
  | 't'
  | 'kg'
  | 'km/h'
  | 'm/s'
  | 'N'
  | 'kN'
  | 'N/(m/s)'
  | 'N/(km/h)'
  | 'kN/(km/h)'
  | 'N/(m/s)²'
  | 'N/(km/h)²'
  | 'kN/(km/h)²';
export const convertUnits = (prevUnit: Unit, newUnit: Unit, prevValue: number): number => {
```

I suggest we create a new type for the unit options to :
- make sure we use the right units (stop typing mistakes)
- have a pretty autocompletion
- make it easier to update, like removing a unit.

Reply:
Would be really nice but it implies that the InputGroupSNCFValue['unit'] needs to be a Unit type as well and the component is also used for allowances which units are different.

Moreover, it also implies that the SchemaParameter with units property get the Unit[] type as well but they can also include the units with the ton (/t) so it gets messy...